app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -12,6 +14,7 @@ class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
     require_nested :Refresher
 
     include AuthenticationMixin
+    include ::HawkularUtilsMixin
```

Comment:
Where does this come from? If not needed, then please add it in a different commit